hw/market.py

```diff
@@ -1,28 +1,40 @@
+from datetime import date
+from decorator import logger
+
+
 class Market:
     def __init__(self, wines: list = None, beers: list = None) -> None:
-        pass
+        self.wines = {}
+        self.beers = {}
+        for elem in wines:
+            self.wines[elem.title] = elem
+        for elem in beers:
+            self.beers[elem.title] = elem
```

Comment:
в питоне можно сделать чуть проще
```suggestion
        self.wines = {drink.title: drink for drink in wines}
        self.beers = {drink.title: drink for drink in beers}
```